src/components/material/MaterialAvailabilityText/MaterialAvailabilityText.tsx

```diff
@@ -12,19 +16,28 @@ interface Props {
 }
 
 const MaterialAvailabilityText: React.FC<Props> = ({ manifestations }) => {
-  return (
-    <>
-      {hasCorrectAccessType(AccessTypeCode.Physical, manifestations) && (
-        <MaterialAvailabilityTextPhysical pids={getAllPids(manifestations)} />
-      )}
-      {hasCorrectAccessType(AccessTypeCode.Online, manifestations) &&
-        getAllIdentifiers(manifestations).length > 0 && (
-          <MaterialAvailabilityTextOnline
-            isbns={getAllIdentifiers(manifestations)}
-          />
-        )}
-    </>
-  );
+  const materialType = head(getMaterialTypes(manifestations));
+
+  if (hasCorrectAccessType(AccessTypeCode.Physical, manifestations)) {
+    const pids = getAllPids(manifestations);
+    return <MaterialAvailabilityTextPhysical pids={pids} />;
+  }
+
+  if (
+    hasCorrectAccessType(AccessTypeCode.Online, manifestations) &&
+    getAllIdentifiers(manifestations).length > 0 &&
+    materialType
+  ) {
+    const isbns = getAllIdentifiers(manifestations);
+    return (
+      <MaterialAvailabilityTextOnline
+        isbns={isbns}
```

Comment:
Since isbns is only used for this particular case you can also skip creating a variable that has no particular value other than occuping a tiny bit of memory and instead use the function directly:
```typescript
      <MaterialAvailabilityTextOnline
        isbns={getAllIdentifiers(manifestations)}